prow/cluster/cluster-issuer.yaml

```
@@ -0,0 +1,18 @@
+---
+# cluster-issuer
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  # Adjust the name here accordingly
+  name: letsencrypt-prod
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v01.api.letsencrypt.org/directory
```

This should be acme-v02 now. cert-manager v0.5.0 does not support v01 endpoints
```suggestion
    server: https://acme-v02.api.letsencrypt.org/directory
```

---

ah thanks, this one was copied from kube-lego :see_no_evil: